src/solver/conv_asm_1x1u_bias_activ_fused.cpp

```diff
@@ -208,6 +207,15 @@ ConvBiasActivAsm1x1U::GetSolution(const FusionContext& context,
     return sol;
 }
 
+float ConvBiasActivAsm1x1U::GetWti(const FusionContext&, const FusionDescription&) const
+{
+    // Negative values mean rough estimation of time
+    // Solvers with positive values are considered first priority
+    // If none found solvers with negative are used, -2.0f meaning a completely unknown value
+    // So lower absolute values mean higher priority
+    return -.5f;
```

Comment:
[Q] Are you pretty sure that ConvBiasActivAsm1x1U is better than all other applicable ones that return wti_unknown?

Reply:
No, but they had these priorities before #2388